sync/datahub/metrichub_glossary.py

```diff
@@ -29,7 +29,11 @@ def _build_metric_dict(metric: MetricHubDefinition) -> Dict:
         metric_content += f"_{metric.description.strip()}_\n\n"
 
     if metric.sql_definition:
-        metric_content += f"SQL Definition:\n```{metric.sql_definition.strip()}```"
+        metric_content += f"SQL Definition:\n```{metric.sql_definition.strip()}```\n\n"
+
+    if metric.statistics and len(metric.statistics) > 0:
```

Comment:
If the `len > 0` it should be bool true
```suggestion
    if metric.statistics:
```